app/src/main/java/com/github/se/cyrcle/ui/parkingDetails/ParkingDetailsScreen.kt

```diff
@@ -80,19 +84,30 @@ fun ParkingDetailsScreen(
       parkingViewModel.selectedParking.collectAsState().value
           ?: return Text(stringResource(R.string.no_selected_parking_error))
   val userSignedIn by userViewModel.isSignedIn.collectAsState(false)
+  val scrollState = rememberScrollState()
   val context = LocalContext.current
+
   // === States for the images ===
   var newParkingImageLocalPath by remember { mutableStateOf("") }
   val showDialog = remember { mutableStateOf(false) }
   val imagesUrls by parkingViewModel.selectedParkingImagesUrls.collectAsState()
   val imagesPaths by parkingViewModel.selectedParkingAssociatedPaths.collectAsState()
   val showDialogImage = remember { mutableStateOf<String?>(null) }
   val showDialogImageDestinationPath = remember { mutableStateOf<String?>("") }
+
+  var ownerReputationScore = 0.0
+  var ownerUsername = stringResource(R.string.undefined_username)
   if (selectedParking.owner != "Unknown Owner" && selectedParking.owner != null) {
-    userViewModel.selectSelectedParkingUser(parkingViewModel.selectedParking.value?.owner!!)
-  } else {
-    userViewModel.setParkingUser(null)
+    userViewModel.getUserById(
```

Comment:
Why did you remove this line? Pretty sure that field needs to be added still

Reply:
Nice, you changed it!

Reply:
I deleted that during a rebase when I shouldnt have.